### DISCUSSION TITLE
Does Rspack have a svg logo ?

### DISCUSSION THREAD

**Discussion by lalalazero** [web-infra-dev/rspack#4845] [Q&A] [answered]
Hello

I'd like to add framework guide at tailwindcss document side.

like this

![image](https://github.com/web-infra-dev/rspack/assets/20458239/6f0295be-148a-4512-90a0-06d1c3759a08)

so I found out that these brand logos are all svg format. 

Does Rspack have a svg logo too ?

**Comment by chenjiahan** [comment]
There is the SVG logo:

- Minified:

![rspack_logo_minified](https://github.com/web-infra-dev/rspack/assets/7237365/4e52e3d6-e80b-4f93-bd24-1c6e220175c3)

- Original:
![rspack_logo_original](https://github.com/web-infra-dev/rspack/assets/7237365/bed1e2de-fa18-43b6-8426-2739b0d44ab3)

**Comment by chenjiahan** [comment]
Look forward to seeing the Rspack logo on the Tailwindcss page. Maybe Rsbuild should be added too 😄

https://rsbuild.dev/guide/basic/tailwindcss

**Comment by lalalazero** [comment]
sure, it's nice to add Rsbuild too. does Rsbuild have a svg logo too ?

**Comment by chenjiahan** [comment]
Of course, please check 😄

- minified
![rsbuild-minified](https://github.com/web-infra-dev/rspack/assets/7237365/977e9f1c-1f5f-4915-9389-0ba0483e9947)

- original

![rsbuild-original](https://github.com/web-infra-dev/rspack/assets/7237365/0343ac3b-4395-4487-967f-74044aba3089)

**Comment by chenjiahan** [comment]
The previous Rspack logo is too big, so I made a smaller one. You can use this:

![rspack_logo (1)](https://github.com/web-infra-dev/rspack/assets/7237365/75828f3c-f3d0-41e8-bdd9-86a65a5df829)

**Comment by chenjiahan** [comment, marked as answer]
Update:

All design resources related to Rsfamily are uploaded to this repo: https://github.com/rspack-contrib/rsfamily-desgin-resources
